src/coreclr/jit/emitfmtsarm64_sve.h

```diff
@@ -0,0 +1,514 @@
+IF_DEF(SVE_2AQ, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AQ
+IF_DEF(SVE_2AR, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AR
+IF_DEF(SVE_2AS, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AS
+IF_DEF(SVE_2AT, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AT
+IF_DEF(SVE_2AU, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AU
+IF_DEF(SVE_2AV, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AV
+IF_DEF(SVE_2AW, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AW
+IF_DEF(SVE_2AX, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AX
+IF_DEF(SVE_2AY, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AY
+IF_DEF(SVE_2AZ, IS_NONE, NONE) // Instruction has 2  possible encoding types, type AZ
+IF_DEF(SVE_2BA, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BA
+IF_DEF(SVE_2BB, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BB
+IF_DEF(SVE_2BC, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BC
+IF_DEF(SVE_2BD, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BD
+IF_DEF(SVE_2BE, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BE
+IF_DEF(SVE_2BF, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BF
+IF_DEF(SVE_2BG, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BG
+IF_DEF(SVE_2BH, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BH
+IF_DEF(SVE_2BI, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BI
+IF_DEF(SVE_2BJ, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BJ
+IF_DEF(SVE_2BK, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BK
+IF_DEF(SVE_2BL, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BL
+IF_DEF(SVE_2BM, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BM
+IF_DEF(SVE_2BN, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BN
+IF_DEF(SVE_2BO, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BO
+IF_DEF(SVE_2BP, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BP
+IF_DEF(SVE_2BQ, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BQ
+IF_DEF(SVE_2BR, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BR
+IF_DEF(SVE_2BS, IS_NONE, NONE) // Instruction has 2  possible encoding types, type BS
+
+/*****************************************************************************
+*           SVE_WW_XY
+*           WW      -- code
+*           X       -- register count
+*           Y       -- unique id
+
+*****************************************************************************/
+
+IF_DEF(SVE_AA_3A,   IS_NONE, NONE) // SVE_AA_3A  ........xx...... ...gggmmmmmddddd  -- sve_int_bin_pred_log
```

Comment:
What is the comment at the end? Does that come from the XML? Is it one example instruction from the XML? How do we read the name?

Reply:
Thanks for pointing it out. I was using the code names and now, using the description. Hope they serve better purpose.